src/SFA.DAS.CandidateAccount.Application/Application/Queries/GetApplicationWorkHistories/GetApplicationWorkHistoriesQueryResult.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace SFA.DAS.CandidateAccount.Application.Application.Queries.GetApplicationWorkHistories;
+
+public record GetApplicationWorkHistoriesQueryResult
+{
+    public List<Domain.Application.WorkHistoryEntity> WorkHistories { get; init; } = [];
```

Comment:
Can we add a mapping to a domain model please so that we aren't directly linking the Entity to the API presentation?

Reply:
Domain Model introduced rather than entities exposed